src/Illuminate/View/ComponentAttributeBag.php

```diff
@@ -116,7 +116,7 @@ public function merge(array $attributeDefaults = [])
             }
 
             if ($key !== 'class') {
-                $attributes[$key] = $attributeDefaults[$key] ?? $value;
+                $attributes[$key] = $value ?? $attributeDefaults[$key];
```

Comment:
Won't this crash whenever `attributeDefaults[$key]` is unset?

Reply:
We probs want this to be `$attributes[$key] = $value ?? $attributeDefaults[$key] ?? $value;`

Reply:
Good point, but it would never be evaluated by virtue of an attribute being specified in the template. 

This block should probably just be:
```php
if ($key !== 'class') {
    $attributes[$key] = $value;

    continue;
}
```

Reply:
The defaults are applied at the bottom of the merge() function